src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px #f3f3f3 solid;
+  border-radius: 5px;
+
+  &:hover {
+    scale: 120%;
+    transition: 0.3s;
```

Comment:
Move transition to parent block. 
If your checking, now card have transition when we hover it. 
But in case when you unhover card animation cuts. 
Moving transition to parent selector can fixed it.